docs/conf_sso_test_msscolab.rst

```diff
@@ -14,79 +14,92 @@ To set up a local identity provider with the mscolab server, you'll first need t
     3. Enable USE_SAML2
     4. Generate Metadata Files
     5. Start the Identity Provider
-    6. Restart the mscolab Server
+    6. Start the mscolab Server
     7. Test the Single Sign-On (SSO) Process
 
 
-Initial Steps
--------------
-Before getting started, you should correctly activate the environments, set the correct Python path, and be in the correct directory (`$ cd MSS`), as explained in the mss instructions : https://open-mss.github.io/develop/Setup-Instructions
+1. Initial Steps
+----------------
+Before getting started, you should correctly activate the environments, set the correct Python path as explained in the mss instructions : https://github.com/Open-MSS/MSS/tree/develop#readme
 
 
 
-Generate Keys, Certificates, and backend_saml files
----------------------------------------------------
+2. Generate Keys, Certificates, and backend_saml files
+------------------------------------------------------
 
-This involves generating both .key files and .crt files for both the Identity provider and mscolab server and backend_saml.yaml file. 
+This involves generating both `.key` files and `.crt` files for both the Identity provider and mscolab server and `backend_saml.yaml` file. 
 
-Before running the command make sure to set
-`USE_SAML2 = False` 
-In some cases, if you set `USE_SAML2 = True` without certificates, this will not execute. So, make sure to set `USE_SAML2 = False` before executing the command.
+Before running the command make sure to set `USE_SAML2 = False` in your `mscolab_settings.py` file,  You can accomplish this by following these steps:
```

Comment:
I've tried to set `USE_SAML2 = True` in ~/mss/mscolab_settings.py, while ~/mss is in `$PYTHONPATH`. That didn't work though, to successfully execute `mscolab sso_conf --init_sso_metadata` I still had to set this in mslib/mscolab/conf.py for some reason.

Do you have this issue as well?

Reply:
The Syntax for develop compared to stable had changed.

It is only one line no class definition

Reply:
Ah, that was it, thanks. I didn't catch that format change.